src/Microsoft.AspNetCore.Authentication.OpenIdConnect/Events/IOpenIdConnectEvents.cs

```diff
@@ -36,6 +36,11 @@ public interface IOpenIdConnectEvents : IRemoteAuthenticationEvents
         Task RedirectToIdentityProviderForSignOut(RedirectContext context);
 
         /// <summary>
+        /// Invoked before redirecting to the post logout redirect uri.
+        /// </summary>
+        Task RedirectToPostLogoutRedirectUri(RedirectContext context);
```

Comment:
If interfaces cause too much versioning pain when adding new events, then removing them to exclusively use the `OpenIdEvents` class might be an interesting option for 2.0 (as long as we keep the delegate properties + virtual methods, of course :smile:).